src/components/hooks/useNotification.js

```diff
@@ -0,0 +1,18 @@
+import { useDispatch } from 'react-redux'
+
+import { showNotification } from 'store/reducers/notificationReducer'
+
+const useNotification = () => {
+  const dispatch = useDispatch()
+  const dispatchNotification = (textMessage, type) => {
+    dispatch(
+      showNotification({
+        text: textMessage,
+        type: type,
+      })
+    )
+  }
+  return dispatchNotification
+}
```

Comment:
et si on en profitait pour ajouter des proxys aux types aussi, genre : 

```suggestion
const useNotification = () => {
  const dispatch = useDispatch()
  const dispatchNotification = (textMessage, type) => {
    dispatch(
      showNotification({
        text: textMessage,
        type: type,
      })
    )
  }
  return {
    success: (textMessage) => dispatchNotification(textMessage, 'success'),
    error: (textMessage) => dispatchNotification(textMessage, 'error'),
    info: (textMessage) => dispatchNotification(textMessage, 'info'),
  }
}
```

l'usage serait : 

```js
const notify = useNotification();
notify.success('message');
```

Reply:
2° se serait bien d'avoir la signature typescript tant qu'on y es
en suivant ton implementation

```ts
type UseNotification = () => (
  textMessage: string, 
  type: 'success' | 'error' | 'infos'
) : void

const useNotification: UseNotification = () => {...}
```

ou en suivant mon exemple 

```ts
type UseNotification = () => {
   success(textMessage: string) : void;
   error(textMessage: string) : void;
   info(textMessage: string) : void;
}

const useNotification: UseNotification = () => {...}
```

Reply:
pas mal :+1: